README.md

```diff
@@ -15,22 +15,42 @@ With the release of AndroidX, Android Support is now considered deprecated and w
 
 ## Binding Policies
 
-- This repository binds over 100 AndroidX libraries that are published to [NuGet.org](https://nuget.org). The full package list can be found in [config.json](config.json).
+- This repository binds over 200 (2022-09) AndroidX libraries that are published to [NuGet.org](https://nuget.org). The full 
```

Comment:
Let's not put in dates.  They imply we either need to constantly update them, or our documentation will look outdated.

Reply:
We had number so I updated the number and added date. I was aware of the timestamp problem, so I did not put in the date, just yyyy-MM indicating - approx date of that info. Also no exact number for the same reason, but gives user feeling about the numbers here (and our effort).

I had intention to add links to Maven and NuGet repos, but it was too much for now.

Reply:
date removed